cli/cmd/backup.go

```diff
@@ -77,8 +81,8 @@ func (c *ServicedCli) cmdBackup(ctx *cli.Context)  {
 		return
 	}
 	// do backup
-	if path, err := c.driver.Backup(args[0], ctx.StringSlice("exclude")); err != nil {
-		fmt.Fprintln(os.Stderr, err)
+	if path, err := c.driver.Backup(args[0], ctx.StringSlice("exclude"), ctx.Bool("force")); err != nil {
+		fmt.Fprintln(os.Stdout, err)
```

Comment:
The message is logged in filesystem.go: https://github.com/control-center/serviced/blob/4412af56901bdbdc601df095ada55d35dfc5631f/dao/elasticsearch/filesystem.go#L109-L111. The code here is for the user.